docs/src/main/tut/cookbook.md

```diff
@@ -351,9 +351,33 @@ val policy: Cors.Policy = Cors.Policy(
 val corsService: Service[Request, Response] = new Cors.HttpFilter(policy).andThen(service)
 ```
 
+### Creating a Finagle Client
+
+Since Finch is built on top of Finagle, it shares its utilities like it's stateless [client][client]. Do read the Finagle's
+documentation to get a full understanding of the configuration options for the client like its [loadbalancing][loadbalancing] abilities. A simple example is given below.
+
+```tut:silent
+import com.twitter.finagle.Http
+import com.twitter.finagle.http._
+import com.twitter.util.{Await, Future}
+
+val host = "api.nasa.gov"
+val client: Service[Request, Response] = Http.client.withTls(host).newService(s"$host:443")
+val request = Request(Method.Get, "/planetary/apod?api_key=NNKOjkoul8n1CH18TWA9gwngW1s1SmjESPjNoUFo")
+val response: Future[Response] = client(request)
+
+Await.result(response.onSuccess { rep: Response =>
+  //println("GET success: " + rep.contentString)
```

Comment:
We might want to remove this commented out line.

Reply:
@rpless Agree on all above will update